Lesson 12/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Lesson 12",
```

```suggestion
  "name": "lesson-12",
```

согласно соглашениям, в названии используется [kebab-case](https://github.com/vladshcherbin/course/blob/master/resources/conventions.md#kebab-case)